cc/mcts_tree.cc

```diff
@@ -709,6 +709,17 @@ Coord MctsTree::SoftPickMove(Random* rnd) const {
   for (size_t i = 0; i < cdf.size(); ++i) {
     cdf[i] = std::pow(root_->child_N(i), options_.policy_softmax_temp);
   }
+
+  // Clear probabilities for moves in pass-alive territory.
```

Comment:
This is thorough but i think we can maybe skip this & the call to CalculatePassAliveRegions:  we do softpick in the first 30 moves.  If there's pass alive territory in the first 30 moves, things are probably pretty busted...

Reply:
done